pkg/ssh/client/ssh.go

```diff
@@ -48,6 +48,24 @@ func LoadKeyFromFile(filePath, clusterName, ngName string, ec2API ec2iface.EC2AP
 	return keyName, nil
 }
 
+func fingerprint(filePath string, key []byte) (string, error) {
+	pk, _, _, _, err := ssh.ParseAuthorizedKey(key)
+	if err != nil {
+		return "", fmt.Errorf("parsing key %q: %w", filePath, err)
+	}
+
+	if strings.Contains(pk.Type(), "ed25519") {
```

Comment:
Ah hang on. This could just be `pk.Type() == ssh.KeyAlgoED25519`.

Reply:
I'll make it more specific